tests/unit/test_service_steps.py

```diff
@@ -661,3 +662,31 @@ def test_emr_modify_instance_group_by_name_step_creation():
         'End': True
     }
 
+
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_databrew_start_job_run_step_creation():
+    step = DataBrewStartJobRunStep('Start Databrew Job Run', parameters={
+        "Name": "MyWorkflowJobRun"
+    })
+
+    assert step.to_dict() == {
+        'Type': 'Task',
+        'Resource': 'arn:aws:states:::databrew:startJobRun.sync',
+        'Parameters': {
+            'Name': 'MyWorkflowJobRun'
+        },
+        'End': True
+    }
+
+    step = DataBrewStartJobRunStep('Start Databrew Job Run', wait_for_completion=False, parameters={
+        "Name": "MyWorkflowJobRun"
+    })
+
+    assert step.to_dict() == {
+        'Type': 'Task',
+        'Resource': 'arn:aws:states:::databrew:startJobRun',
+        'Parameters': {
+            'Name': 'MyWorkflowJobRun'
+        },
+        'End': True
+    }
```

Comment:
nit: can we split these tests out into 2 tests instead? I know this is probably what we've done for the others, but I think tests should only try and evaluate one scenario

Reply:
Agreed - will add the changes in the next commit